services/static-webserver/client/source/class/osparc/data/Resources.js

```diff
@@ -1337,6 +1337,9 @@ qx.Class.define("osparc.data.Resources", {
 
           if ([404, 503].includes(status)) {
             message += "<br>Please try again later and/or contact support";
+          } else if (status == 500) {
+            console.error(message);
+            message += "<br>Please contact support";
```

Comment:
All 500 now have a proper message from the backend. I would not add "contact support"